plugins/add_file_folder_to_sidebar.py

```diff
@@ -0,0 +1,114 @@
+import sublime
+import sublime_plugin
+
+import os
+
+# Related reading:
+#     https://stackoverflow.com/q/71255440/814803
+#
+# This plugin is based on a request made in the above stack overflow question
+# for a method by which you can add the path of the current file to the side
+# bar.
+#
+# This plugin will allow for that, and supports versions of Sublime Text all
+# the way back to version 2 (which is the version the question asker uses). It
+# defines a command that will examine the path of the currently active file and
+# then, if it's not already open in the side bar, add it.
+#
+# To use this in Sublime Text 2, you need to adjust the value of the _subl_path
+# variable to point to your Sublime Text 2 executable; for ST3 and above the
+# plugin will use the native plugin API instead.
+#
+# To use the plugin from the context menu, create a file named
+# Context.sublime- menu in your User package and give it the following content
+# (or add the command to an existing file, if you have one).
+#
+# [
+#     { "caption": "-", "id": "file" },
+#     { "command": "add_file_folder_to_side_bar", "caption": "Add Folder to Side Bar",}
+# ]
+#
+# If you prefer a key binding, you can add one such as the following:
+#
+# { "keys": ["ctrl+alt+a"], "command": "add_file_folder_to_side_bar"},
+
+
+# This needs to point to the "sublime_text" executable for your platform; if
```

Comment:
Did Sublime Text 2 not have a `subl` command?

Reply:
The comment should also mention that this is required for ST2 only.